plugins/kubernetes/test/models/watchers/deploy_watcher_test.rb

```diff
@@ -9,6 +9,9 @@
   let(:project) { current_release.project }
 
   before do
+    # Disable multithreading so celluloid stays in the same transaction
+    ActiveRecord::Base.stubs(connection: ActiveRecord::Base.connection)
```

Comment:
nono ... only send PRs ;)

On Wed, Apr 6, 2016 at 9:15 PM, Roman Sandler <notifications@github.com>
wrote:

> In plugins/kubernetes/test/models/watchers/deploy_watcher_test.rb
> <https://github.com/zendesk/samson/pull/791#discussion_r58817362>:
>
> > @@ -9,6 +9,9 @@
> >    let(:project) { current_release.project }
> >
> >    before do
> > +    # Disable multithreading so celluloid stays in the same transaction
> > +    ActiveRecord::Base.stubs(connection: ActiveRecord::Base.connection)
>
> hmm that blogger sounds familiar, should we report the soft_deletion issue
> upstream to him as well?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/791/files/219ed5c94ed281ccefa4a06d38f003bc956e7d70#r58817362>
>